cookbooks/logstash/recipes/default.rb

```diff
@@ -1,16 +1,37 @@
 # Logstash recipe
 
 # Create directories
-directory "#{node["logstash"]["dir"]}"
-directory "#{node["logstash"]["dir"]}/conf"
-directory "#{node["logstash"]["dir"]}/data"
+directory "#{node["logstash"]["dir"]}" do
```

Comment:
Implemented, will appear in next pull request.
I am using strings for hash keys/parameters, for consistency with the rest of the code. :ok_hand: